src/herbivoresAndCarnivores.js

```
@@ -1,15 +1,33 @@
 'use strict';
-
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  hide() {
+    this.hidden = true;
+  }
```

Herbivores should have `hidden` defaulting to `false` until `hide()` is called. The current `Herbivore` implementation does not initialize `hidden` (it will be `undefined`), but the task examples and description expect herbivores to show `hidden: false` before `hide` is called. Please initialize `hidden` to `false` for newly created Herbivore instances (for example, provide a constructor that calls `super(name, health)` then sets `this.hidden = false`). See the task description examples for the expected initial state of herbivores.